packages/backend/server/src/core/auth/service.ts

```diff
@@ -263,6 +263,46 @@ export class AuthService implements OnApplicationBootstrap {
     });
   }
 
+  async refreshSessionIfNeeded(
+    _req: Request,
+    res: Response,
+    token: string,
+    ttr = this.config.auth.session.ttr
+  ): Promise<boolean> {
+    const session = await this.getSession(token);
+    if (!session) {
+      return false;
+    }
+
+    if (
+      session.expiresAt &&
```

Comment:
`Session` is a list of `UserSession`, and `Session.expiresAt` currently is always `null`.